packages/cli/src/commands/inspect.ts

```diff
@@ -67,12 +87,28 @@ export default async function main(client: Client) {
   // extract the first parameter
   let [, deploymentIdOrHost] = argv._;
 
-  if (argv._.length !== 2) {
+  if (!deploymentIdOrHost) {
+    // if the URL is not passed in, check stdin
+    // allows cool stuff like `vc inspect $(vc deploy --prod --no-wait) --wait`
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏change example

I like allowing for the url to be piped in, but this specific example doesn't seem useful. Is there a reason to do:

```
vc inspect $(vc deploy --prod --no-wait) --wait
```

over

```
vc deploy --prod
```

?

```suggestion
    // allows cool stuff like `echo my-deployment.vercel.app | vc inspect --wait`
```

Reply:
Fixed. I guess I was just excited that it worked. :)